build.gradle

```diff
@@ -13,92 +13,95 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 buildscript {
+  //we define kotlin version for benefit of both core and test (see kotlin-gradle-plugin below)
   ext.kotlinVersion = '1.3.71'
   repositories {
 	maven { url "https://repo.spring.io/plugins-release" }
   }
   dependencies {
-	classpath 'org.springframework.build.gradle:propdeps-plugin:0.0.7'
-	classpath 'io.spring.gradle:spring-io-plugin:0.0.4.RELEASE'
-	classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.0'
+	classpath 'org.springframework.build.gradle:propdeps-plugin:0.0.7' //still uses the old ways
 	classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlinVersion}"
   }
 }
 
 
 plugins {
-	id 'org.asciidoctor.convert' version '1.5.9.2'
+	id "com.github.johnrengelman.shadow" version "4.0.2"
```

Comment:
do we really need top-level `shadow`? I don't even think we shade anything in 3.2.x

Reply:
mmh maybe we can remove it from this commit and keep it only in 3.3.x ? I _think_ we need it for either benchmarks or reactor-tools, don't we?

Reply:
yeah, I think this is indeed only needed for 3.3